pkg/lib/esbuild-eslint-plugin.js

```diff
@@ -6,12 +6,14 @@ import { ESLint } from 'eslint';
 
 const NAME = 'eslintPlugin';
 
-export const eslintPlugin = {
+export const eslintPlugin = ({
+    filter = /src\/.*\.(jsx?|js?)$/,
```

Comment:
Don't we need `(src|pkg)`?

Reply:
will do! This is overwritable so I was just passing the filter variable again when this was called from this repository. But your suggestion also works.

Reply:
Hmm, could this be flipped around? Can we ignore node_modules/ and include all other *.js and *.jsx files?